Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/entity/OutboundDocRetrieveStrategyBase.java

```diff
@@ -41,7 +42,7 @@
 import ihe.iti.xds_b._2007.RetrieveDocumentSetRequestType.DocumentRequest;
 import ihe.iti.xds_b._2007.RetrieveDocumentSetResponseType;
 import java.util.List;
-
+import gov.hhs.fha.nhinc.common.auditlog.*;
```

Comment:
Can you use the individual import(s) here instead of bringing in the whole package?